source/server/domain/process.coffee

```diff
@@ -27,7 +27,10 @@ class Space.eventSourcing.Process extends Space.eventSourcing.Aggregate
 
   trigger: (command) ->
     command.meta ?= {}
-    command.meta[this.eventCorrelationProperty] = this.getId()
+    if this.getId.toString?
```

Comment:
this check is not necessary as any string-like object should have `toString` defined and you can call it on any string without problems.